pr_agent/settings/configuration.toml

```diff
@@ -100,6 +101,21 @@ polling_interval_seconds = 30
 # token to authenticate in the patch server
 # patch_server_token = ""
 
+
```

Comment:
This doesn't belong here! Should be placed by every user in their .sercrets.toml file